amundsen_application/static/js/components/common/SearchBar/index.tsx

```diff
@@ -109,33 +100,9 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
     this.setState({ showTypeAhead: false });
   }
 
-  isFormValid = (searchTerm: string) : boolean => {
-    if (searchTerm.length === 0) {
-      return false;
-    }
-
-    const hasAtMostOneCategory = searchTerm.split(':').length <= 2;
-    if (!hasAtMostOneCategory) {
-      this.setState({
-        subText: SYNTAX_ERROR_CATEGORY,
-        subTextClassName: ERROR_CLASSNAME,
-      });
-      return false;
-    }
-
-    const colonIndex = searchTerm.indexOf(':');
-    const hasNoSpaceAroundColon = colonIndex < 0 ||
-      (colonIndex >= 1 && searchTerm.charAt(colonIndex+1) !== " " &&  searchTerm.charAt(colonIndex-1) !== " ");
-    if (!hasNoSpaceAroundColon) {
-      this.setState({
-        subText: `${SYNTAX_ERROR_PREFIX}'${searchTerm.substring(0,colonIndex).trim()}:${searchTerm.substring(colonIndex+1).trim()}'${SYNTAX_ERROR_SPACING_SUFFIX}`,
-        subTextClassName: ERROR_CLASSNAME,
-      });
-      return false;
-    }
-
-    this.setState({ subText: SUBTEXT_DEFAULT, subTextClassName: "" });
-    return true;
+  isFormValid = () : boolean => {
+    const form = document.getElementById("search-bar-form") as HTMLFormElement;
+    return form.checkValidity();
```

Comment:
It can, depending on what rules we set for the `SearchBar`. As of this upload it will stop submission if the pattern is not matched or the search bar is empty. Unsure whether these rules will apply in the final version, but if not then the method will be removed